lib/elixir/lib/kernel.ex

```diff
@@ -4020,24 +4020,30 @@ defmodule Kernel do
       is allowed and its results will be evaluated at runtime. Usually
       evaluates to the name of a module.
 
+      If the expression evaluates to a list, it will emit a warning,
+      since lists have been deprecated.
+
     * `:as` - the function to call on the target given in `:to`.
       This parameter is optional and defaults to the name being
       delegated (`funs`).
 
+  ### Deprecated options
+
+  The following option has been deprecated and will emit a warning if used.
```

Comment:
We say "Deprecated options" in the type but then say "The following option has been deprecated". Maybe let's say "The following options have been deprecated".